src/components/footer.tsx

```diff
@@ -0,0 +1,12 @@
+const Footer = () => {
+  return (
+    <div className="flex w-screen flex-col items-center gap-5 px-10">
```

Comment:
```suggestion
    <div className="flex w-screen flex-col items-center px-10">
```
add padding on the bottom instead of gap, pb class